client-java/controller/src/main/java/org/evomaster/client/java/controller/internal/db/SqlHandler.java

```diff
@@ -149,7 +149,11 @@ public ExecutionDto getExecutionDto() {
 
         buffer.stream()
                 .forEach(sql -> {
-                    if (isSelect(sql) || isDelete(sql) || isUpdate(sql)) {
+                    /*
+                        for `SELECT 1`, the dist is always 0, might skip to compute heuristics for it
+                        check with Andrea
+                     */
+                    if (!isSelectOne(sql) && (isSelect(sql) || isDelete(sql) || isUpdate(sql))) {
```

Comment:
hi. should be fine. i guess any operation with no `WHERE` should be skipped. but i guess `SELECT 1` might be the only one

Reply:
could add this clarification as a comment

Reply:
With the latest commit, I add a comment regarding the method ie, `getDistances`. is it correct?